x/ibc/04-channel/keeper/keeper_test.go

```diff
@@ -65,6 +72,7 @@ func (suite *KeeperTestSuite) SetupTest() {
 	suite.chainB = NewTestChain(testClientIDB)
 
 	suite.cdc = suite.chainA.App.Codec()
+	suite.querier = ibckeeper.NewQuerier(*suite.chainA.App.IBCKeeper)
 }
```

Comment:
querier should be moved be field on each chain, will update in a testing refactor pr